modules/sdk-core/src/bitgo/utils/tss/baseTSSUtils.ts

```diff
@@ -154,6 +157,26 @@ export default class BaseTssUtils<KeyShare> extends MpcUtils implements ITssUtil
     throw new Error('Method not implemented.');
   }
 
+  /**
+   * Signs a transaction using TSS MPCv2 for ECDSA and through utilization of custom share generators
+   *
+   * @param {TSSParams | TSSParamsForMessage} params - params object that represents parameters to sign a transaction or a message.
+   * @param {CustomMPCv2Round1GeneratingFunction} externalSignerMPCv2Round1Generator - a function that creates MPCv2 Round 1 shares in the ECDSA TSS MPCv2 flow.
+   * @param {CustomMPCv2Round2GeneratingFunction} externalSignerMPCv2Round3Generator - a function that creates MPCv2 Round 2 shares in the ECDSA TSS MPCv2 flow.
+   * @param {CustomMPCv2Round3GeneratingFunction} externalSignerMPCv2Round3Generator - a function that creates MPCv2 Round 3 shares in the ECDSA TSS MPCv2 flow.
+   * @param {RequestType} requestType - the type of the request to sign (transaction or message).
+   * @returns {Promise<TxRequest>} - a signed tx request
+   */
+  signEcdsaMPCv2TssUsingExternalSigner(
+    params: TSSParams | TSSParamsForMessage,
+    externalSignerMPCv2Round1Generator: CustomMPCv2Round1GeneratingFunction,
+    externalSignerMPCv2Round2Generator: CustomMPCv2Round2GeneratingFunction,
+    externalSignerMPCv2Round3Generator: CustomMPCv2Round3GeneratingFunction,
+    requestType?: RequestType
+  ): Promise<TxRequest> {
+    throw new Error('Method not implemented.');
```

Comment:
This still needs to be implemented in this PR right? Or will this be addressed in a different PR?

Reply:
Nop, the base utils use the non implemented errors incase you used the wrong util (like eddsa)